application/service/factory/service_factory.go

```diff
@@ -159,6 +170,10 @@ func NewServiceFactory(producer ServiceContextProducer, config *configuration.Co
 	f.notificationServiceFunc = func() service.NotificationService {
 		return notificationservice.NewNotificationService(f.getContext(), f.config)
 	}
+	// default function to return an instance of Cluster Service
+	f.clusterServiceFunc = func() service.ClusterService {
+		return clusterservice.NewClusterService(f.getContext(), config)
```

Comment:
`f.config` would be better than `config`

Reply:
Done in https://github.com/fabric8-services/fabric8-auth/pull/659/commits/7db39812d4b5e9743dff65a22c87ce9660cf8946